rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc

```diff
@@ -23,7 +23,9 @@ include::modules/osd-aws-privatelink-firewall-prerequisites.adoc[leveloffset=+1]
 [role="_additional-resources"]
 .Additional resources
 
-*  xref:../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring]
+* xref:../../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring]
+* xref:rosa-security-groups_prerequisites[Security groups]
```

Comment:
You have an incomplete path for this xref.

Reply:
```suggestion
* xref:../../rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc#rosa-security-groups_prerequisites[Security groups]
```